tests/src/main/java/net/neoforged/neoforge/debug/data/registries/DatapackEntryTests.java

```diff
@@ -46,8 +45,8 @@ static void conditionalDatapackEntries(final DynamicTest test, final Registratio
                 event.getLookupProvider(),
                 builder,
                 conditions -> {
-                    conditions.accept(CONDITIONAL_FALSE_DAMAGE_TYPE, FalseCondition.INSTANCE);
-                    conditions.accept(CONDITIONAL_TRUE_DAMAGE_TYPE, TrueCondition.INSTANCE);
+                    conditions.accept(CONDITIONAL_FALSE_DAMAGE_TYPE, IConditionBuilder.of().never());
```

Comment:
I was thinking that too, but that would have to wait until 21.
Since afaik cant make static versions cause the method signature will conflict the the default ones.